cmd/cluster/health.go

```diff
@@ -221,18 +261,24 @@ func ocmDescribe(clusterID string) ([]string, string, int, error) {
 	cluster := response.Body()
 	cloudProvider := cluster.CloudProvider().ID()
 	cloudProviderMessage := strings.ToUpper(cloudProvider)
+	clusterInfo.Name = cluster.Name()
 
 	healthObject.ID = cluster.ID()
 	healthObject.Name = cluster.Name()
 	healthObject.Provider = cloudProviderMessage
+	healthObject.AZs = cluster.Nodes().AvailabilityZones()
+	healthObject.Expected.Infra = cluster.Nodes().Infra()
+	healthObject.Expected.Master = cluster.Nodes().Master()
 
 	if cloudProvider != "aws" {
-		return nil, "", 0, fmt.Errorf("This command is only supported for AWS clusters. The command is not supported for %s clusters.", cloudProviderMessage)
-
+		return clusterInfo, fmt.Errorf("This command is only supported for AWS clusters. The command is not supported for %s clusters.", cloudProviderMessage)
 	}
-	totalNodes := cluster.Nodes().Compute() + cluster.Nodes().Infra() + cluster.Nodes().Master()
+	clusterInfo.AZs = cluster.Nodes().AvailabilityZones()
+	clusterInfo.Nodes.Compute = cluster.Nodes().Compute()
+	clusterInfo.Nodes.AutoscaleMin = cluster.Nodes().AutoscaleCompute().MinReplicas()
+	clusterInfo.Nodes.AutoscaleMax = cluster.Nodes().AutoscaleCompute().MaxReplicas()
```

Comment:
There seems to be some duplication between the `healthObject` and `clusterInfo` objects.  Is there a reason to have both separated and not just merge them into one?

Reply:
clusterInfo is just a variable holding the output of ocmDescribe(o.k8sclusterresourcefactory.ClusterID) command and it needs to be separate from healthObject which is what we output because the raw input needs a bunch of preprocessing, the clsuetrStats object is the one that can be merged with ClusterHealthCondensedObject. I cleaned up the code.

Reply:
I removed the clusterInfo variable since it wasn't really necessary to introduce it.